v-next/hardhat/test/internal/cli/main.ts

```diff
@@ -88,12 +77,9 @@ describe("main", function () {
         const cliArguments = command.split(" ").slice(2);
         const usedCliArguments = [false, false, false];
 
-        const res = parseTaskAndArguments(
-          cliArguments,
-          usedCliArguments,
-          hre,
-        ) as TaskRes;
+        const res = parseTaskAndArguments(cliArguments, usedCliArguments, hre);
 
+        assert.ok(!Array.isArray(res));
```

Comment:
same here

Reply:
and in the other `assert.ok`s of the file

Reply:
done